--- gst-plugins-base-0.10.6/ext/libvisual/visual.c.orig	2006-03-10 19:00:00.000000000 +0100
+++ gst-plugins-base-0.10.6/ext/libvisual/visual.c	2006-05-12 20:27:25.060772000 +0200
@@ -697,7 +697,7 @@
       return FALSE;
 
   list = visual_actor_get_list ();
-  for (i = 0; i < visual_list_count (list); i++) {
+  for (i = 0; i < list->count; i++) {
     VisPluginRef *ref = visual_list_get (list, i);
     VisActorPlugin *actplugin = NULL;
     VisPluginData *visplugin = NULL;
@@ -721,16 +721,16 @@
     if (ref->info->plugname == NULL)
       continue;
 
-    actplugin = VISUAL_PLUGIN_ACTOR (visplugin->info->plugin);
+    actplugin = VISUAL_ACTOR_PLUGIN (visplugin->info->plugin);
 
     /* Ignore plugins that only support GL output for now */
-    if (actplugin->depth == VISUAL_VIDEO_DEPTH_GL) {
+    if (actplugin->vidoptions.depth == VISUAL_VIDEO_DEPTH_GL) {
       GST_DEBUG ("plugin %s is a GL plugin (%d), ignoring",
-          ref->info->plugname, actplugin->depth);
+          ref->info->plugname, actplugin->vidoptions.depth);
       is_gl = TRUE;
     } else {
       GST_DEBUG ("plugin %s is not a GL plugin (%d), registering",
-          ref->info->plugname, actplugin->depth);
+          ref->info->plugname, actplugin->vidoptions.depth);
     }
 
     visual_plugin_unload (visplugin);
--- gst-plugins-base-0.10.6/configure.ac.orig	2006-04-28 12:25:33.000000000 +0200
+++ gst-plugins-base-0.10.6/configure.ac	2006-05-12 20:30:01.478547500 +0200
@@ -420,7 +420,7 @@
 dnl *** libvisual ***
 translit(dnm, m, l) AM_CONDITIONAL(USE_LIBVISUAL, true)
 GST_CHECK_FEATURE(LIBVISUAL, [libvisual visualization plugins], libvisual, [
-  PKG_CHECK_MODULES(LIBVISUAL, libvisual = 0.2.0, HAVE_LIBVISUAL="yes", HAVE_LIBVISUAL="no")
+  PKG_CHECK_MODULES(LIBVISUAL, libvisual-0.4 >= 0.4.0, HAVE_LIBVISUAL="yes", HAVE_LIBVISUAL="no")
   AC_SUBST(LIBVISUAL_CFLAGS)
   AC_SUBST(LIBVISUAL_LIBS)
 ])
