--- gst-plugins-base-0.10.20/configure.ac.orig	2008-10-13 03:01:03.048685758 +0200
+++ gst-plugins-base-0.10.20/configure.ac	2008-10-13 03:01:32.046692790 +0200
@@ -508,7 +508,7 @@
 AG_GST_CHECK_FEATURE(THEORA, [Xiph Theora video codec], theora, [
   dnl this check will work as long as theora uses 1.0.x or similar for the
   dnl first post-alpha release and not just 1.0
-  AG_GST_PKG_CHECK_MODULES(THEORA, theora >= 1.0alpha5)
+  AG_GST_PKG_CHECK_MODULES(THEORA, theora >= 1.0RC1)
 ])
 
 dnl *** vorbis ***
