--- gst-plugins-base-1.20.1/meson.build.orig	2022-03-14 12:33:40.000000000 +0100
+++ gst-plugins-base-1.20.1/meson.build	2022-04-16 20:39:18.235165721 +0200
@@ -452,7 +452,8 @@ pkgconfig_variables = ['exec_prefix=${pr
     'datadir=${datarootdir}',
     'girdir=${datadir}/gir-1.0',
     'typelibdir=${libdir}/girepository-1.0',
-    'libexecdir=${prefix}/libexec']
+    'libexecdir=' + join_paths(get_option('prefix'), get_option('libexecdir'))
+]
 pkgconfig_subdirs = ['gstreamer-1.0']
 
 meson_pkg_config_file_fixup_script = find_program('scripts/meson-pkg-config-file-fixup.py')
